### PR TITLE
ZMT New Features & README

### PR DESCRIPTION
**Pull request by connectbo**
This pr includes all features shown in the demo, along with a README file, where I draft some introduction to ZMT and how to use it.

### CODE REVIEW

**Comment by connectbo** [comment]
Conflict resolved.

File: src/views/User.js [hunk 1]
```diff
@@ -129,7 +129,8 @@ function User() {
                 }
             }).then(res => {
                 console.log(res);
-                window.location.reload();
+                updateContent();
+                //window.location.reload();
```
Comment (trel) [inline src/views/User.js]: dead?

**Review by trel — COMMENTED** [review]
which commits are for filtering the list pages?

which add counts to user listing in sidebar?

**Comment by trel** [comment]
lots of codacy things - let's get that count down a bit before merging

**Comment by connectbo** [comment]
Sure. There are some dead code and some just for testing purpose. Will add more commits to clean it.

**Comment by connectbo** [comment]
Added a footer, and it looks like this. React will first read the git SHA and then run the app. Version number is a temporary one.

Screen Shot 2021-02-25 at 9 43 02 AM

**Comment by trel** [comment]
I like the footer - SHA is key, excellent.

File: src/App.js [hunk 2]
```diff
@@ -9,21 +10,26 @@ import Resource from './views/Resource';
...
+            <User path="/user" />
+            <EditGroup path='group/edit' />
+            <Group path='/group' />
+            <Resource path='/resource' />
+            <Logout path='/logout' />
+            <Authenticate path="/" />
+          </Router>
+        </div>
+        <hr />
+        <div className="app_footer"><div><img className="app_footer_logo" src={logo}></img>iRODS Consortium © 2021</div><div>Zone Management Tool Version: 0.1.0, {process.env.REACT_APP_GIT_SHA.substring(0,7)}</div></div>
```
Comment (trel) [inline src/App.js:32]: the version number (0.1.0) should come from a variable?

'iRODS Consortium' should be a link to https://irods.org

File: src/views/Home.js [hunk 3]
```diff
@@ -267,7 +267,6 @@ function Home() {
                             </Collapse>
                         </Card>
                     ) : <div><CircularProgress /> Loading...</div>}</div></main>
-                <footer>iRODS Consortium</footer>
             </div> : <div className={classes.logout}><BlockIcon /><br /><div>Please <a href="http://localhost:3000/">login</a> to use the administration dashboard.</div></div>
```
Comment (trel) [inline src/views/Home.js:271]: what is this?   when does this show?

**Review by trel — COMMENTED** [review]
this is a lot of stuff - which issues is it related to?   

talking about cb160388ccdf5c43c054437608bab2cd7f0889f0

**Comment by trel** [comment]
Going to merge this and start on a new set of issues.

**Pull request merged**